examples/objective-c/hello_world/ViewController.m

```diff
@@ -123,6 +126,20 @@ - (void)addResponseMessage:(NSString *)message
   [self.tableView reloadData];
 }
 
+- (void)sendStats {
+  Element *eleFoo = [[Element alloc] initWithStringLiteral:@"foo"];
+  Element *eleBar = [[Element alloc] initWithStringLiteral:@"bar"];
+  Element *eleCounter = [[Element alloc] initWithStringLiteral:@"counter"];
+  Element *eleGauge = [[Element alloc] initWithStringLiteral:@"gauge"];
+  id<Counter> counter = [self.statsClient counterWithElements:@[ eleFoo, eleBar, eleCounter ]];
+  [counter incrementWithCount:1];
+
+  id<Gauge> gauge = [self.statsClient gaugeWithElements:@[ eleFoo, eleBar, eleGauge ]];
+  [gauge setWithValue:1];
+  [gauge addWithAmount:1];
+  [gauge subWithAmount:1];
```

Comment:
Why are these different from Android?

Reply:
you mean the method names? 

they come from the swift generated `Envoy/Envoy-Swift.h`

Reply:
I meant the amounts

Reply:
to clarify: you mean the actual numbers sent (instead of the method names)